.github/workflows/sync-issue-with-pr.yaml

```
@@ -78,8 +84,4 @@ jobs:
               fi
             done <<< "$LABELS"
           
-          done
-
-
-
-# todo remove major, minor, and patch labels before applying this workflow
\ No newline at end of file
+          done
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)